web/locales/en/messages.ftl

```diff
@@ -763,20 +767,40 @@ want-to-continue = Do you want to continue?
 finish-editing = Finish editing first?
 lose-changes-warning = Leaving now means you’ll lose your changes
 build-custom-goal = Build a custom goal
-help-reach-hours = Help reach { NUMBER($hours) } hours in { $language } with a personal goal
-help-reach-hours-general = Help Common Voice reach { NUMBER($hours) } hours in a language with a personal goal
+help-reach-hours = Help reach { NUMBER($hours) ->
```

Comment:
So, unfortunately, you're not going to be able to just edit an existing string and you have to create new IDs for them instead and change those ID values in the components (you can delete the original IDs). That's because languages that have e.g. `help-reach-hours` already localized will not know that the original English source text has changed, and these edited strings will not show up in Pontoon as needing re-translation. I know it's inelegant, but it's a limitation of the system. I'd suggest just adding `-pluralized` to the end or something to indicate the difference between the original and this one.